### PR TITLE
Added first version Swedish translations

### PR DESCRIPTION
**Pull request by ExodusActual**
## Description

I have added a first version of the pages in Swedish  some of the link descriptions are not translated. 
I have not added functionality to select the Swedish language on the page. Just added the translation files.

## Related Issue

 Issue #222 

## Checklist:

- [ x] I have read the contributing guidelines in the project README.
- [ x] My code follows the code style of this project.
- [ x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by ExodusActual** [comment]
Not sure why it fails or how to fix it. Sorry, just did the translations.

**Comment by samajammin** [comment]
This is amazing! Thank you for your contribution.

I'm sorry for the inconvenience but would you mind adding this translation to our CrowdIn page? This makes it easiest for out translation team to process. Here's an invite to get started:
https://crowdin.com/project/ethereumfoundation/invite

Here's the project:
https://crowdin.com/project/ethereumfoundation/sv-SE#

Once that's complete, I can fix these build errors & get this deployed. Feel free to post any questions you have here. I can connect you with the lead of our Translation Program to ensure you have everything you need.

Looking forward to getting your Swedish translations up on the website!

File: docs/sv/beginners/index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: Ethereum för nybörjare
```
Comment (sorumfactory) [inline docs/sv/beginners/index.md]: This is another sample example.

**Comment by samajammin** [comment]
@ExodusActual if you don't mind granting me access to your fork of this repo, I can work to fix the failing build & complete this PR. Please let me know.

**Comment by samajammin** [comment]
@ExodusActual FYI I'm going to take these source files & create a new PR to fix the issues. I'll tag you on it.

**Comment by samajammin** [comment]
Closing this out in favor of #398 - thanks again @ExodusActual!

**Comment by samajammin** [comment]
@all-contributors please add @ExodusActual for translation

**Pull request closed without merging**